src/locale/locales/zh-CN.json

```diff
@@ -40,7 +40,7 @@
     "version": "0HFoHIB1i+ZETBO3YI76/g=="
   },
   "SNTable.Pagination.RowsPerPage": {
-    "value": "每页行数",
+    "value": "每页行数：",
```

Comment:
Hi @ashrles, we would like to remove : here

Reply:
will make an bugfix PR for this issue